podcasts/Sharing/Clip/PlayButton.swift

```diff
@@ -0,0 +1,19 @@
+import SwiftUI
+
+struct PlayButton: View {
```

Comment:
Maybe rename it to `TrimPlayButton`?

I can see confusion when searching for the current `PlayPauseButton` in the full player.